src/prefect/cli/docker-compose.yml

```diff
@@ -13,12 +14,21 @@ services:
       - ${POSTGRES_DATA_PATH}:/var/lib/postgresql/data
     networks:
       - prefect-server
-    restart: "always"
     command:
       - "postgres"
       # explicitly set max connections
       - "-c"
       - "max_connections=150"
+    # Use `pg_isready` to check the connection status of the PostgreSQL server
+    healthcheck:
+      test: pg_isready -q -d $${POSTGRES_DB} -U $${POSTGRES_USER} || exit 1
+      interval: 5s
+      timeout: 2s
+      retries: 18
+      start_period: 1s
```

Comment:
I am testing this locally.

Reply:
I set this to a low number because I didn't want to impact starting time to much, however, starting time will depend both on `start_period` and having a small `interval` if the first test should fail.

What would you think is a more reasonable number for `start_period`?

Reply:
I'm inclined to set the interval even lower, for example, 1 second, this speed up deployment, however with the overhead of having more healthchecks (which should be minimal)

Reply:
I think 1s for repeats would be reasonable. Perhaps 2s for the database start period. We can always adjust this in the future :)

Reply:
Made most healthchecks (besides UI) run every 1s